pkg/daemon/ceph/client/info.go

```diff
@@ -42,6 +42,7 @@ type ClusterInfo struct {
 	CephVersion   cephver.CephVersion
 	Namespace     string
 	OwnerInfo     *k8sutil.OwnerInfo
+	RequireMsgr2  bool
```

Comment:
Do we need a change to `LoadClusterInfo()` to fill in this field?

Reply:
`LoadClusterInfo()` doesn't have access to the cluster CR, so this field is set sequentially in the reconcile after the basic info is loaded.